src/components/PodcastCard/PodcastCard.js

```diff
@@ -1,21 +1,48 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import { useState } from 'react'
 
 PodcastCard.propTypes = {
   image: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
+  author: PropTypes.string,
+  description: PropTypes.string,
+  categories: PropTypes.object,
 }
 
-export default function PodcastCard({ image, title }) {
+export default function PodcastCard({
+  image,
+  title,
+  author,
+  description,
+  categories,
+}) {
+  const [expanded, setExpanded] = useState(false)
+
   return (
-    <Card>
-      <Image
-        src={image}
-        alt={`Podcast Picture of ${title}`}
-        width="150px"
-        height="150px"
-      />
-      <Title>{title}</Title>
+    <Card onClick={() => setExpanded(!expanded)} clicked={expanded}>
+      <CardMainContainer clicked={expanded}>
+        <Image
+          src={image}
+          alt={`Podcast Picture of `}
+          width="150px"
+          height="150px"
+        />
+        <InnerContainer>
+          <Title>{title}</Title>
+          <Author clicked={expanded}>by {author}</Author>
```

Comment:
I would not expect that `clicked` sets `display: none` by reading this line only.
A property like `<Author hidden={expanded}>` could make more sense.

Instead of setting `display: none`, I would prefer to not render the element at all:
```suggestion
          {!expanded && <Author>by {author}</Author>}
```